lms/static/js/spec/learner_dashboard/course_entitlement_view_spec.js

```diff
@@ -22,16 +22,16 @@ define([
             initialSessionId = isAlreadyEnrolled ? testSessionIds[0] : '';
             entitlementAvailableSessions = [{
                 enrollment_end: null,
-                session_start: '2013-02-05T05:00:00+00:00',
+                start: '2013-02-05T05:00:00+00:00',
                 pacing_type: 'instructor_paced',
                 session_id: testSessionIds[0],
-                session_end: null
+                end: null
             }, {
                 enrollment_end: '2017-12-22T03:30:00Z',
-                session_start: '2018-01-03T13:00:00+00:00',
+                start: '2018-01-03T13:00:00+00:00',
                 pacing_type: 'self_paced',
                 session_id: testSessionIds[1],
-                session_end: '2018-03-09T21:30:00+00:00'
+                end: '2018-03-09T21:30:00+00:00'
```

Comment:
This date isn't really that far out. Could this timebomb like we just had in mktg?

Reply:
More specifically, will some tests start to fail months after we pass 2018-03-09?

Reply:
It won't fail but worth changing